vueui/src/collections.js

```diff
@@ -1,271 +0,0 @@
-/**
```

Comment:
I think it's safe to assume that this file can be removed, but is this the only change that's not just auto-formatting / style change? If you can guarantee that that'd simplify the review quite a bit.

Reply:
Yes, it's safe to remove, it was used by darg.js, what was replaced by byWin.js